book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] It would be nice to have a setter for rentals to follow the project requirements to form the many-to-many relationship completely 👍 So it would be better if you if you have `attr_accessor :rentals` 😊